content/admin/policy_mgmt/Sensitive-data-masking.md

```diff
@@ -0,0 +1,61 @@
+<!--
+title: "Sensitive Data Masking"
+description: "How to manage sensitive data masking"
+tags: "admin policy management sensitive data masking"
+-->
+
+Take advantage of Contrast's data masking feature to limit risk to your organization and meet compliance requirements.  
+
+## How It Works
+
+Contrast's data masking feature protects sensitive data in your applications by preventing the Contrast agent from sending it to external systems: the Contrast UI, syslog or the security log. Data masking can also protect information by preventing the agent from sending it as part of Contrast vulnerability or attack reports. 
+
+Sensitive data masking is part of policy management by default for organizations with at least one active Assess or Protect license. Contrast offers several categories of sensitive data, or data types, that are comprised of specific keywords that the agent automatically identifies and redacts in reports. 
+
+### Agent identification and masking
+
+Contrast agents mask sensitive data in query parameters, request headers, cookies and forms. Your agent identifies sensitive data by searching for specific keywords used in the input name. If the agent finds a match, it redacts the value for that input, and replaces it with a placeholder with the format `contrast-redacted-{datatype}`, where `datatype` is the category of sensitive data to which the keyword belongs.
```

Comment:
Should this say "body" instead of "forms"? Or is forms ok? @seschis

Reply:
yea, body is more correct given the surrounding terms.  But the only body type we are currently parsing for fine-grained masking is a form-post body, so I can see why the term was used here.